outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "s3_trail_bucket_name" {
```

Comment:
Let's just call it `bucket_name` since there is no intention to have more than one bucket.